vertexai/src/LiveGenerativeModel.cs

```diff
@@ -71,7 +71,7 @@ internal LiveGenerativeModel(FirebaseApp firebaseApp,
   private string GetURL() {
     return "wss://firebasevertexai.googleapis.com/ws" + 
            "/google.firebase.vertexai.v1beta.LlmBidiService/BidiGenerateContent" +
-           $"/locations/{_location}" +
+           $"/locations/{_backend.Location}" +
```

Comment:
backend could have a null location for googleAI, we currently might want to do some grace failure to throw exception earlier if the instantiation of the LiveModel is with googleAI backend.

Reply:
Good catch. Made it throw an exception if trying to use the LiveModel with GoogleAI for now, until the support is added.